cucumber-expressions/java/src/test/java/io/cucumber/cucumberexpressions/EnumParameterTypeTest.java

```diff
@@ -2,22 +2,28 @@
 
 import org.junit.Test;
 
+import java.util.List;
+import java.util.Locale;
+
 import static java.util.Collections.singletonList;
 import static org.junit.Assert.assertEquals;
 
 public class EnumParameterTypeTest {
-    @Test
-    public void constructs_enum() {
-        ParameterType<Color> t = new ParameterType<>("color", "\\w+", Color.class, new Transformer<Color>() {
-            @Override
-            public Color transform(String arg) {
-                return Color.valueOf(arg);
-            }
-        });
-        assertEquals(Color.BLUE, t.transform(singletonList("BLUE")));
+    public enum Mood {
+        happy,
+        meh,
+        sad
     }
 
-    public enum Color {
-        RED, BLUE, YELLOW
+    @Test
+    public void converts_to_enum() {
+        ParameterTypeRegistry registry = new ParameterTypeRegistry(Locale.ENGLISH);
+        registry.defineParameterType(ParameterType.fromEnum(Mood.class));
```

Comment:
Is this a parameter type *from*Enum or *for*Enum?

When using a static import I think `registry.defineParameterType(forEnum(Mood.class));`  reads a bit better.

Reply:
OTOH: fromEnum is kinda the well known standard.